### PR TITLE
v1.15.0

### PR DESCRIPTION
**Pull request by vrundakansara**
### Description
* New: FAQ - Schema
* New: How-To Schema
* Improvement: Advanced Columns : UI Improvements (Column Gap option).
* Fix: Advanced Columns - Active state of Vertical alignment option not visible in the editor.
* Fix: Icon List - Accessibility issue when link is enabled.
* Fix: Post Grid - Multiple AJAX requests generated if Pagination Markup is returned empty.
* Fix: Post Grid - Pagination not calculating properly in few cases.
* Fix: 404 error for Google Fonts in some cases issue fixed.
* Fix: Table of Contents - Bullets not showing issue fixed.

### Types of changes
New feature (non-breaking change which adds functionality)

### Checklist:
- [x] My code is tested
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-admin-helper.php [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+		}
+
+		/**
+		 * Returns an option from the database for
+		 * the admin settings page.
+		 *
+		 * @param  string  $key     The option key.
+		 * @param  mixed   $default Option default value if option is not available.
+		 * @param  boolean $network_override Whether to allow the network admin setting to be overridden on subsites.
+		 * @return string           Return the option value
```
Comment (RajkiranBagal) [inline classes/class-uagb-admin-helper.php:46]: since tag missing

File: classes/class-uagb-admin-helper.php [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+			return apply_filters( 'uagb_enabled_blocks', UAGB_Helper::$block_list );
+		}
+
+		/**
+		 * Updates an option from the admin settings page.
+		 *
+		 * @param string $key       The option key.
+		 * @param mixed  $value     The value to update.
+		 * @param bool   $network   Whether to allow the network admin setting to be overridden on subsites.
+		 * @return mixed
```
Comment (RajkiranBagal) [inline classes/class-uagb-admin-helper.php:97]: since tag missing

File: classes/class-uagb-block-helper.php [hunk 3]
```diff
@@ -16,6 +16,130 @@
...
+
+			$selectors[' .uagb-tools__wrap .uagb-how-to-tools-child__wrapper'] = array(
+				'color' => $attr['subHeadingColor'],
+			);
+
+			$selectors[' .uagb-how-to-materials .uagb-how-to-materials-child__wrapper'] = array(
+				'color' => $attr['subHeadingColor'],
+			);
+
+			$t_selectors = array();
```
Comment (RajkiranBagal) [inline classes/class-uagb-block-helper.php:125]: Double declaration, Declared the same on line 32 and 33

File: classes/class-uagb-config.php [hunk 4]
```diff
@@ -699,86 +874,87 @@ public static function get_block_attributes() {
...
-							'classMigrate'            => false,
-							'align'                   => 'center',
+							'block_id' => '',
+							'height'   => '300',
+						),
+					),
+					'uagb/how-to'             => array(
+						'slug'        => '',
+						'title'       => __( 'How-to Schema', 'ultimate-addons-for-gutenberg' ),
+						'description' => __( 'How To Configure How-To Schema In Ultimate Addons For Gutenberg?', 'ultimate-addons-for-gutenberg' ),
```
Comment (RajkiranBagal) [inline classes/class-uagb-config.php]: Is the The Description of How-To schema intended like this?

File: classes/class-uagb-block-helper.php (hunk 3 above)
Comment (vrundakansara) [reply classes/class-uagb-block-helper.php]: Removed the double declaration. 👍

File: classes/class-uagb-admin-helper.php (hunk 2 above)
Comment (vrundakansara) [reply classes/class-uagb-admin-helper.php:98]: Added.

File: classes/class-uagb-admin-helper.php (hunk 1 above)
Comment (vrundakansara) [reply classes/class-uagb-admin-helper.php:46]: Added the tag.

File: classes/class-uagb-config.php (hunk 4 above)
Comment (vrundakansara) [reply classes/class-uagb-config.php]: The updated description is 
```
This block allows you to design attractive How-to pages or articles with automatically adding How-to Schema to your page.
```

**Pull request merged**